webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -356,28 +356,28 @@ class Boot extends Loggable {
 
     // All the following is related to the sitemap
     val nodeManagerMenu =
-      (Menu("NodeManagerHome", <i class="fa fa-sitemap"></i> ++ <span>Node management</span>: NodeSeq) /
+      (Menu("10", <i class="fa fa-sitemap"></i> ++ <span>Node management</span>: NodeSeq) /
```

Comment:
I think we should keep the number only as a prefix, like what you did for plugin below, because: 
- it will make debugging in some cases easier (IIRC, we have a map of menu id -> menu at some places, and having only "10" won't help much)
- having globally unique name is less risky if at some point we make a map of all menu items (including sub menu).

Reply:
these name will become part of the api (since they can be used by plugin), we need to normalize them, I propose: 
- two digits followed by  `-`
- lower case, no space, `-` between words

Reply:
OK, using a name is perhaps not super futur proof, because it will be hard to rename that id. I'm not sur it is of importance (we can still rename between major version, since plugin are not compatible between them).